### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by krschau**
## Summary of the pull request
Update readme to include up-to-date information about installation and .vsconfig

## References and relevant issues
https://task.ms/44327242

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,13 +94,17 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
...
 
 1. Clone the repository
-2. Configure your system, please use the [configuration file](.configurations/configuration.dsc.yaml). This can be applied by either:
-   * Dev Home's machine configuration tool
-   * WinGet configuration. If you have WinGet version [v1.6.2631 or later](https://github.com/microsoft/winget-cli/releases), run `winget configure .configurations/configuration.dsc.yaml` in an elevated shell from the project root so relative paths resolve correctly
+2. Configure your system
+   * Please use the [configuration file](.configurations/configuration.dsc.yaml). This can be applied by either:
+     *  Dev Home's machine configuration tool
+     * WinGet configuration. If you have WinGet version [v1.6.2631 or later](https://github.com/microsoft/winget-cli/releases), run `winget configure .configurations/configuration.dsc.yaml` in an elevated shell from the project root so relative paths resolve correctly
+   * Alternatively, if you already are running the minimum OS version, have Visual Studio installed, and have developer mode enabled, you may configure your Visual Studio directly via the .vsconfig file. To do thhis:
```
Comment (hamnarauf) [inline README.md:101]:
```suggestion
   * Alternatively, if you already are running the minimum OS version, have Visual Studio installed, and have developer mode enabled, you may configure your Visual Studio directly via the .vsconfig file. To do this:
```
Small typo.

**Review by adrastogi — APPROVED** [review]
:shipit:

File: README.md [hunk 2]
```diff
@@ -85,7 +85,6 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
 * [Kayla Cinnamon](https://github.com/cinnamon-msft), Product Manager: [@cinnamon_msft](https://twitter.com/cinnamon_msft)
 * [Clint Rutkas](https://github.com/crutkas), Senior Product Manager: [@clintrutkas](https://twitter.com/clintrutkas)
```
Comment (cinnamon-msft) [inline README.md:86]:
```suggestion
* [Clint Rutkas](https://github.com/crutkas), Principal Product Manager: [@clintrutkas](https://twitter.com/clintrutkas) 
```

File: README.md [hunk 3]
```diff
@@ -85,7 +85,6 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
 * [Kayla Cinnamon](https://github.com/cinnamon-msft), Product Manager: [@cinnamon_msft](https://twitter.com/cinnamon_msft)
```
Comment (cinnamon-msft) [inline README.md:85]:
```suggestion
* [Kayla Cinnamon](https://github.com/cinnamon-msft), Senior Product Manager: [@cinnamon_msft](https://twitter.com/cinnamon_msft)
```

File: README.md [hunk 4]
```diff
@@ -95,13 +94,17 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
 ## Building the code
 
 1. Clone the repository
-2. Configure your system, please use the [configuration file](.configurations/configuration.dsc.yaml). This can be applied by either:
-   * Dev Home's machine configuration tool
-   * WinGet configuration. If you have WinGet version [v1.6.2631 or later](https://github.com/microsoft/winget-cli/releases), run `winget configure .configurations/configuration.dsc.yaml` in an elevated shell from the project root so relative paths resolve correctly
+2. Configure your system
+   * Please use the [configuration file](.configurations/configuration.dsc.yaml). This can be applied by either:
+     *  Dev Home's machine configuration tool
```
Comment (cinnamon-msft) [inline README.md:99]:
```suggestion
     * Dev Home's machine configuration tool
```

**Review by cinnamon-msft — APPROVED** [review]
LGTM :)

**Pull request merged**